lms/static/sass/course/_info.scss

```diff
@@ -229,5 +229,13 @@ div.info-wrapper {
         }
       }
     }
+
+    @media print {
```

Comment:
This ensures that the "Handouts" sidebar on the course info page does not print. Although it may contain useful information, it typically runs over the margin and off the page, and generally doesn't print nicely.